src/ServiceMonitor/IISConfigUtil.cpp

```diff
@@ -214,7 +214,7 @@ HRESULT IISConfigUtil::RunCommand(wstring& pstrCmd, BOOL fIgnoreError)
         //
         // appcmd command failed
         //
-        _tprintf(L"\nAPPCMD failed with error code %d\n", dwStatus);
+        _tprintf(L"\n[SERVICEMONITOR] APPCMD failed with error code %d\n", dwStatus);

```

Comment:
Why we add "[SERVICEMONITOR]" everywhere in the log? Is it intended to differentiate the logs from servicemonitor VS from other sources? If so, it should be a feauture of LogMonitor to support "tags" or "labels" from different sources instead of us specifing it in the unstructured verbose logs.

Reply:
Good point. Updated as suggested.